SimulationAgent/DeviceProperties/Tag.cs

```diff
@@ -1,52 +1,50 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
 using System;
-using System.Threading.Tasks;
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services;
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Diagnostics;
-using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Models;
 
-namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.SimulationAgent.DeviceConnection
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.SimulationAgent.DeviceProperties
 {
     /// <summary>
     /// Add twin information to the new device twin
     /// </summary>
-    public class DeviceTwinTag : IDeviceConnectionLogic
+    public class Tag : IDevicePropertiesLogic
```

Comment:
could use a longer name, this is a bit ambiguous, e.g. `TagDevice` or `SetDeviceTag` etc. By the way, if you don't want to use the term "twin" I assume you should also rename the event `DeviceTwinTagged` right ?